src/apps/relay/userdb.c

```diff
@@ -711,6 +727,14 @@ void release_allocation_quota(uint8_t *user, int oauth, uint8_t *realm) {
     ur_string_map_unlock(rp->status.alloc_counters);
     free(username);
   }
+
+#ifndef _MSC_VER
+  global_allocation_count--;
+  TURN_LOG_FUNC(TURN_LOG_LEVEL_INFO, "Global turn allocation count decremented, now %ld\n", global_allocation_count);
```

Comment:
This can be INFO when the server is in drain mode - I think it will be useful to see in logs that the draining is happening...
But not while in regular operation. We already have too much logging.

Reply:
Makes sense, I'll make a change for that.